src/utils.ts

```diff
@@ -0,0 +1,59 @@
+const OK = 200
+const METHOD_NOT_ALLOWED = 405
+const BED_REQUEST = 400
+
+export const readRequestBody = (req: {
+    on: (
```

Comment:
Why do you type this in such a strange manner? Is req not just of type http `Request`?